index.html

```diff
@@ -2512,22 +2507,20 @@ <h3>WoT Scripting API</h3>
                 time-critical logic from the cloud down to a gateway or
                 edge node.
             </p>
-            <p> The non-normative <a>WoT Scripting API</a> document defines
-                the structure and algorithms of the programming
-                interface that allows scripts to discover, process
-                (retrieve) and produce (define) <a>Thing Descriptions</a>. The
-                implementation of the <a>WoT Scripting API</a> instantiates local
+            <p> The non-normative <a>WoT Scripting API</a> document [[?wot-scripting-api]
+                defines the structure and algorithms of the programming
+                interface that allows scripts to discover, fetch and process,
```

Comment:
There is no fetch() in Scripting API any more.

Reply:
it is about the functionality in general. Would "retrieve" make you happier?

Reply:
There must be a functionality to reference a TD by URI, no?

Reply:
OK, since fetch() is there in the scripting runtime actually, in the general sense we can leave fetch there if it's not explicitly said it's a requirement for the API itself. From the current text I deduce this is a mandatory requirement for the Scripting API, which should not be - but it's a mandatory requirement to the Scripting Runtime (one of its global objects exposed to the Scripting API).

Reply:
When the WoT Scripting API relies on a fetch() funtion in the runtime, it should make it part of the standardized API. Thus, it would again be part of the WoT Scripting API, even when not implemented by WoT-specific objects.

Reply:
No, the Scripting TF disagrees with you on this one (will provide link to the calls where this has been discussed). Fact is that fetch() is not any more part of the Scripting API.